src/components/formio/textarea.tsx

```diff
@@ -0,0 +1,97 @@
+import clsx from 'clsx';
+import {Field, useFormikContext} from 'formik';
+import {useContext, useRef} from 'react';
+
+import {RenderContext} from '@/context';
+import CharCount from '@/utils/charcount';
+import {ErrorList, useValidationErrors} from '@/utils/errors';
+
+import Component from './component';
+import Description from './description';
+import {withMultiple} from './multiple';
+
+export interface TextAreaProps {
+  name: string;
+  label?: React.ReactNode;
+  required?: boolean;
+  tooltip?: string;
+  description?: React.ReactNode;
+  showCharCount?: boolean;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+export const TextArea: React.FC<JSX.IntrinsicElements['textarea'] & TextAreaProps> = ({
+  name,
+  label,
+  required = false,
+  tooltip = '',
+  description = '',
+  showCharCount = false,
+  onChange,
+  ...props
+}) => {
+  const {getFieldProps, getFieldMeta} = useFormikContext();
+  const {value, onChange: formikOnChange} = getFieldProps<string | undefined>(name);
+  const {touched} = getFieldMeta<string | undefined>(name);
+  const {errors, hasErrors} = useValidationErrors(name);
+  // const [{value}, {touched}] = useField<string | undefined>(name);
```

Comment:
Is this left here as todo?

Reply:
nope, it's probably left over debugger stuff. I'll remove it.